_includes/master/reqs-sys.md

```diff
@@ -41,13 +41,17 @@ Ensure that your hosts and firewalls allow the following traffic.
 | flannel networking (VXLAN)                                   | All                 | Bidirectional   | UDP 4789 |
 | etcd datastore                                               | etcd hosts          | Incoming        | [Officially](http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt) TCP 2379 but can vary |
 | {{site.prodname}} networking, cluster of more than 50 nodes  | Typha agent hosts   | Incoming        | TCP 5473 (default) |
-| All                                                          | kube-apiserver host | Incoming        | Often TCP 443 or 6443: check the `targetPort` value returned by `kubectl get svc kubernetes -o yaml` |
+| All                                                          | kube-apiserver host | Incoming        | Often TCP 443 or 6443\* |
 {%- else %}
 | All                                                          | etcd hosts          | Incoming        | [Officially](http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt) TCP 2379 but can vary |
 {%- endif %}
 {%- if include.orch == "OpenShift" %}
-| All                                                          | kube-apiserver host | Incoming        | Often TCP 443 or 8443: check the `targetPort` value returned by `kubectl get svc kubernetes -o yaml` |
+| All                                                          | kube-apiserver host | Incoming        | Often TCP 443 or 8443\* |
```

Comment:
8443 -> 6443.

I think we should say that the default value is 6443, as per the [kube-apiserver docs](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)

Reply:
It is 8443 for OpenShift....
We have 6443 in the K8s section